src/components/Card.jsx

```diff
@@ -21,26 +21,32 @@ export default function Card({
         {image_source ? (
           <img className='card-image' src={image_source} alt={title} />
         ) : (
-          <img
-            className='card-image'
-            src={emptyImage}
-            alt={`비어있는 ${title} 이미지`}
-          />
+          <img className='card-image' src={emptyImage} alt={`비어있는 ${title} 이미지`} />
         )}
       </div>
       <button className='add-bookmark-icon'>
         <img src={StarImg} alt='즐겨찾기 추가' />
       </button>
       <div className='card-info'>
-        <p className='card-log-time'>{logTime}</p>
+        <div className='card-time-and-kebab'>
+          <p className='card-log-time'>{logTime}</p>
+          <div className='kebab-container'>
+            <button
+              className='kebab-icon'
+              onClick={(e) => {
+                e.preventDefault();
+                setViewDropBox(!viewDropBox);
```

Comment:
```suggestion
                setViewDropBox(prev => !prev);
```
렌더링이 2번 일어나면서 다시 마운트 되는 부분 혹시 이 부분을 이렇게 변경해 보시겠어요?